README.md

```diff
@@ -92,6 +92,30 @@ http-server --proxy http://localhost:8080?
 
 Note the `?` at the end of the proxy URL. Thanks to [@houston3](https://github.com/houston3) for this clever hack!
 
+## Using SSL
+
+First, make sure that you have `key.pem` and `cert.pem` files. You can generate them using this command:
+
+``` sh
+openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem
```

Comment:
Is `openssl` built in to all operating systems? If not, we should mention how to install it, or provide a link, or at the very least mention that it is a requirement.

Reply:
Finished